features/full_tests/batch_1/internal_error_reports.feature

```diff
@@ -1,7 +1,8 @@
 Feature: Cached Error Reports
 
 Scenario: If an exception is thrown when sending errors/sessions then internal error reports should be sent
-    When I run "InternalErrorScenario"
+    When I set the screen orientation to portrait
```

Comment:
I think ideally it'd be good to get to the bottom of why the previous scenario is left as landscape. If this test gets removed or skipped then I can see this causing all sorts of head scratching about what's going on

Reply:
I tracked this down to the behavior of the Appium background_app routine changing between 1.8.0 and 1.9.1.  The app appears landscape when coming back to the foreground after the timeout period.

I've updated the tests now so that the orientation is correcting in the scenario that it goes awry in and added a comment.  I'll also raise a an issue on Appium's github.